packages/datetime/src/dateRangePicker.tsx

```diff
@@ -100,6 +100,14 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
      */
     shortcuts?: boolean | IDateRangeShortcut[];
 
+    /**
+     * Whether to show only a single month calendar
+     * If `true`, will only show a single month calendar
+     * If `false`, will show two sequential calendars side-by-side
```

Comment:
remove the two `if` lines, just the "Whether..." is enough (plus trailing period)

> Whether to show only a single month calendar.